src/main/java/edu/escuelaing/ieti/matchmaking/model/Pago.java

```diff
@@ -0,0 +1,35 @@
+package edu.escuelaing.ieti.matchmaking.model;
+
+import java.util.Date;
+
+public class Pago {
```

Comment:
Please have a unified coding convention as a group. Do not mix english and spanish when defining entity names.